lib/application.js

```diff
@@ -127,28 +131,8 @@ module.exports = class Application extends Emitter {
     };
   }
 
-  /**
-   * Initialize a new context.
-   *
-   * @api private
-   */
-
   createContext(req, res) {
```

Comment:
Instead of keeping `createContext` around you could replace where it's used [in the http callback](https://github.com/simplyianm/koa/blob/33467bf49d00a145ba5172d99d3ed9718ff4e108/lib/application.js#L128) with the call to the constructor.